lisa/sut_orchestrator/qemu/schema.py

```diff
@@ -14,16 +14,30 @@ class CloudInitSchema:
     # Additional values to apply to the cloud-init user-data file.
     extra_user_data: Optional[str] = None
 
+@dataclass_json()
+@dataclass
+class LibvirtHost:
+    address: str = ""
+    username: str = ""
+    private_key_file: str = ""
 
 # QEMU orchestrator's global configuration options.
 @dataclass_json()
 @dataclass
 class QemuPlatformSchema:
+    # An optional remote host for the VMs. All test VMs will be spawned on the
+    # specified host by connecting remotely to the libvirt instance running on it.
+    # If None, the local machine is used as host.
+    host: Optional[LibvirtHost] = None
```

Comment:
`hosts`

Reply:
What's the use-case for multiple hosts?

Reply:
It makes the platform scalable for testing. In our experience with Hyper-V platform in LISAv2, there are cases. 1) it needs more than one host to test faster, 2) some scenarios need to check live migrations, move vm from one machine to another one, and some tests need to test the traffic across hosts.

Reply:
This might be a lot of work to support. So, feel free to place a restriction that `hosts` can only contain `1` item for now. Then in the future, we can do the work to add support for multiple hosts.

Reply:
> This might be a lot of work to support. So, feel free to place a restriction that `hosts` can only contain `1` item for now. Then in the future, we can do the work to add support for multiple hosts.

It sounds a plan, let's make the schema as final version firstly.

Reply:
Added schema support for multiple hosts.